tests/test_tpp_backend.py

```diff
@@ -48,6 +48,13 @@ def set_database_url(monkeypatch):
         monkeypatch.setenv("DATABASE_URL", os.environ["TPP_DATABASE_URL"])
 
 
+@pytest.fixture
```

Comment:
It seems more complicated than necessary to make this a fixture as I doubt it will be reused. I think you can just make the test function accept `monkeypatch` and then fiddle with DATABASE_URL there.

Reply:
Yes, you're right, the way I did this is left over from an approach I threw away.